core/src/test/java/cucumber/runtime/formatter/HTMLFormatterTest.java

```diff
@@ -737,8 +753,13 @@ private void runFeaturesWithFormatter(URL outputDir) throws Throwable {
         stepsToLocation.put("first step", "path/step_definitions.java:3");
         hooks.add(TestHelper.hookEntry("after", result("passed")));
         hooks.add(TestHelper.hookEntry("after", result("passed")));
-        hookActions.add(createEmbedHookAction("fakedata".getBytes("US-ASCII"), "image/png"));
-        hookActions.add(createEmbedHookAction("dodgy stack trace here".getBytes("US-ASCII"), "text/plain"));
+        if (isNamedEmbedding) {
```

Comment:
i think it would be better to remove `isNamedEmbedding` flag
as we can have both embeddings with and without name in one report.

Reply:
I deleted this flag and updated existing tests, so they cover case with and without name.